.vscode/settings.json

```diff
@@ -23,7 +23,7 @@
       "splitTerminals": [
         {
           "name": "firebase",
-          "commands": ["firebase emulators:start"],
+          "commands": ["firebase emulators:start --import ./export"],
```

Comment:
lets name the export file something more meaningful. e.g. "emulator_test_config" (assuming that's what is in it, which I think it is...).

Reply:
fixed in 9a95290e5d04a9f824d3501641fb11e7b25e52ff